### DISCUSSION TITLE
Software Pentagon for more 128k?

### DISCUSSION THREAD

**Discussion by tuFFo126** [UzixLS/zx-sizif-512#36] [General]
Where to look for programs on Sizif Pentagon that use more memory than 128k.  What I found after BadApple nothing works.  do you have something?

**Comment by UzixLS** [comment]
Wild Player (including DivMMC-hacked version)

**Reply by tuFFo126** [reply]
Any more programs?  Can Sizif with 512k really handle the right Pentagon128k programs only?

**Reply by UzixLS** [reply]
I'm not using any other above-128K software, so cant tell anything. You can ask i.e. on zx-pk.ru for it.

**Reply by tuFFo126** [reply]
WildPlayer, even when I started with divIDE to have the whole 512k, it showed that the ram is 256k.
WildPlayer also starts without a problem in 128 mode.

**Reply by tuFFo126** [reply]
I'm from Poland and there may be a problem with zx-pk.ru But I will try.

**Reply by UzixLS** [reply]
WildPlayer doesn't show amount of RAM in its UI, at least I've never seen this. With only 128K it just set border to red, with any amount above there is black border. 256K required to get playlist mode working ('0' key).

**Reply by tuFFo126** [reply]
![20230428_160743](https://user-images.githubusercontent.com/124886433/235170286-2794d8d1-bd22-4b11-b767-375fdf532386.jpg)
In mode divMMC

**Reply by UzixLS** [reply]
Ah ok. Probably it just doesnt use anything above 256K.
![изображение](https://user-images.githubusercontent.com/1556813/235173445-aa3e5945-56ce-4a66-befc-af5e6d5ed6b9.png)
